### DISCUSSION TITLE
Micronbar font issue after resizing fonts in MTEX settings file

### DISCUSSION THREAD

**Discussion by JasonOttUW** [mtex-toolbox/mtex#1298] [Ask Anything] [answered]
I recently edited the MTEX settings file to change the default font size, which was very small due to being set based on the screen resolutions (from the ppi variable being used as the basis for the fontSize setting). This worked great for making all the text visible in my pole figures, ebsd map plots, etc with the exception of the text on the micron scale-bar. I was happy with this being resized along with all the other text, but now the transparent grey background on the micronbar does not span the text, and the text itself tends to get cut off by the figure boundary (based on the location setting of the micronbar, I imagine). I have attached an example of this issue below. I am unsure how I can fix this issue--does anyone have any suggestions?

Thanks, 
Jason
![GB15_02A_mis2mean](https://user-images.githubusercontent.com/82618787/144924367-21e24e52-ad2d-4570-a308-b32d10e0518b.png)

**Comment by kilir** [comment, marked as answer]
Hi, easiest would be to change the font size of the micronbar or the length of the displayed units.
```
f=gcm;
mp=getappdata(f.currentAxes,'mapPlot');
mp.micronBar.txt.FontSize = 16; % change font size
%or
mp.micronBar.length = 500; % change length
```

Cheers,
Rüdiger

**Reply by AdamLeaghj** [reply]
Thanks Rudiger!

**Comment by vitesh13** [comment]
Hi,
I find this suggestion a bit difficult to implement. Where do these commands go, when we are plotting an IPF map?
Here is my example code and now where do I put these commands?
```
f = gcf;
ipfKey = ipfColorKey(ebsd_rot('Aluminum'));
ipfKey.inversePoleFigureDirection = vector3d.X;
colors = ipfKey.orientation2color(ebsd_rot('Aluminum').orientations);
plot(ebsd_rot('Aluminum'),colors);
```

**Reply by kilir** [reply]
After plotting.

**Reply by vitesh13** [reply]
Yes I got it!
Thanks! it works now!
